docs/src/prototypes/chatMessages/ControlMessages/index.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from 'react'
+import { Chat, ChatItemProps, ShorthandCollection, Avatar, Divider } from '@stardust-ui/react'
+import GroupControlMessages from './GroupControlMessages'
+import ControlMessage from './ControlMessage'
+
+const janeAvatar = {
+  image: 'public/images/avatar/small/ade.jpg',
+  status: {
+    color: 'green',
+    icon: 'check',
+  },
+}
+
+const ChatExample = () => {
+  const items: ShorthandCollection<ChatItemProps> = [
+    {
+      message: {
+        content: (
+          // Adding control message
+          <ControlMessage
+            icon={true}
+            message={{
+              content: (
```

Comment:
We can avoid the content here after the @layershifter's refactoring of the factories. You can immediately use the <div> element inside the message :)

Reply:
hmm, will try